resources/kubernetes/operator/helm/values.yaml

```diff
@@ -310,10 +335,14 @@ collectors:
               - "k8s.pod.ip"
               - "k8s.pod.uid"
               - "k8s.pod.start_time"
+              - "k8s.container.name"
```

Comment:
I think this is not needed. Can you confirm?
I will also double check

Reply:
For the logs pipeline, this field will be provided by the `filelog` receiver container operator.

Reply:
@gizas you are right removed https://github.com/elastic/opentelemetry/pull/24/commits/58860b3376667fa644fef2af8ba6e7d0a96220eb